env.example

```diff
@@ -143,14 +143,18 @@ ETHERPAD_SKIN_VARIANTS="super-light-toolbar super-light-editor light-background
 # LDAP authentication (for more information see the Cyrus SASL saslauthd.conf man page)
 #
 
-# LDAP url for connection
-#LDAP_URL=ldaps://ldap.domain.com/
+# LDAP url:port for connection
+#LDAP_URL=ldaps://ldap.domain.com:636
 
 # LDAP base DN. Can be empty
```

Comment:
It would be nice the explain the effect of using anymous bind vs authenticated bind

Reply:
Added L:163:165